IQKeyBoardManager/IQKeyboardManager.m

```diff
@@ -1535,8 +1552,8 @@ -(NSArray*)responderViews
     //Otherwise fetching all the siblings
     else
     {
-        NSArray *textFields = [_textFieldView responderSiblings];
-        
+        NSArray *textFields = [_textFieldView.parentView responderSiblings];
```

Comment:
I don't think library should adopt this. It will break most of the users code/logic.